### ISSUE TITLE
Release library does not send any events

### ISSUE THREAD

**Issue by monkeber** [getsentry/sentry-native#215]
Hi, I currently experience issues with release library on Windows. SDK seems like sending requests but they do not appear in Sentry at all.
If I link to Debug library - everything works perfectly and events appear in Sentry, if I link to Release - nothing is being sent and folder for sentry db contains only folder with name "00000000-0000-0000-0000-000000000000", while debug version creates something like this: "1e33f629-8ebe-4ea9-7bbb-206651458677".
I have built library with crashpad and without crashpad backend - nothing has changed.
I was able to retrieve logs of the sdk and for both (Debug and Release) everything seems the same:
Debug logs:
```
[sentry] executing task on worker thread
[sentry] sending request using winhttp to https://sentry.io:443/api/5018340/store/:
x-sentry-auth:Sentry sentry_key=73129e8c1890467a8d35063a83e803de, sentry_version=7, sentry_client=sentry.native/0.2.2
content-type:application/json
content-length:19893

[sentry] Received Response:
HTTP/1.1 200 OK
Connection: keep-alive
Date: Wed, 01 Apr 2020 13:10:09 GMT
Content-Length: 41
Content-Type: application/json
Server: nginx
Vary: Origin
access-control-expose-headers: x-sentry-rate-limits, retry-after, x-sentry-error
x-envoy-upstream-service-time: 1
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
```
Release logs:
```
[sentry] executing task on worker thread
[sentry] sending request using winhttp to https://sentry.io:443/api/5018340/store/:
x-sentry-auth:Sentry sentry_key=73129e8c1890467a8d35063a83e803de, sentry_version=7, sentry_client=sentry.native/0.2.2
content-type:application/json
content-length:18503

[sentry] Received Response:
HTTP/1.1 200 OK
Connection: keep-alive
Date: Wed, 01 Apr 2020 13:01:40 GMT
Content-Length: 41
Content-Type: application/json
Server: nginx
Vary: Origin
access-control-expose-headers: retry-after, x-sentry-error, x-sentry-rate-limits
x-envoy-upstream-service-time: 2
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
```

Maybe I'm missing something?

**Comment by Swatinem** [comment]
> folder for sentry db contains only folder with name "00000000-0000-0000-0000-000000000000"

That is definitely wrong. Maybe our random / uuid generation code is broken when doing release builds. I will check that.

Also: When you post the logs here, pleas make sure to remove/obfuscate your `sentry_key` :-D Because now everyone could be uploading stuff to your project. Please generate a fresh DSN, and keep that in mind for next time ;-)

**Comment by monkeber** [comment]
Would be great, will wait for updates.

Thank you for warning, will do.

**Comment by cammm** [comment]
this was probably fixed by #202 have you tried the latest from master ?

**Comment by monkeber** [comment]
Everything is working now indeed with the latest master, I was using 0.2.2 release when it happened, thank you.
